tests/mapping/test_BaseMappingTransformMixin.py

```diff
@@ -77,34 +77,44 @@ def test_pd_series_map_call(self, mocker):
 
         x.transform(df)
 
-        assert spy.call_count == 2, "unexpected number of calls to pd.Series.map"
+        assert spy.call_count == 2, "unexpected number of calls to pd.Series.replace"
 
         call_args = spy.call_args_list[0]
 
         call_pos_arg = call_args[0]
         call_kwargs = call_args[1]
 
-        assert call_kwargs == {}
+        assert call_kwargs == {"regex": False}
```

Comment:
I can't see where this is coming from, please could you add a comment? (or tell me if it's obvious)

Reply:
I don't totally understand this either.  False is the default value for the regex arg in Series.replace so this will be giving us the behaviour we want, juts have no idea where it gets inserted as a kwarg. Have added a comment to highlight this https://github.com/lvgig/tubular/pull/135/commits/01c9f65da452797503b586c357a055dfb27034f4

Reply:
Will be something to do with the way df. replace is implemented vs map

Reply:
Ok, cheers :)